charts/etcd/templates/etcd-statefulset.yaml

```diff
@@ -132,22 +140,25 @@ spec:
         - --insecure-transport=false
         - --insecure-skip-tls-verify=false
         - --endpoints=https://{{ .Values.name }}-local:{{ .Values.etcd.clientPort }}
-        # enable TLS on backup-restore server reusing etcd cert bundle
-        - --server-cert=/var/etcd/ssl/server/tls.crt
-        - --server-key=/var/etcd/ssl/server/tls.key
 {{ else }}
         - --insecure-transport=true
         - --insecure-skip-tls-verify=true
         - --endpoints=http://{{ .Values.name }}-local:{{ .Values.etcd.clientPort }}
 {{- end }}
         - --etcd-connection-timeout={{ .Values.backup.etcdConnectionTimeout }}
```

Comment:
Why not a check for this too?

Reply:
For `etcdConnectionTimeout`, we take a default value from values.yaml if absent.